masschallenge/impact-api
# [AC-5096] - API Request - Expose "Functional Expertise" for Experts via API

**Changes Introduced in [AC-5096](https://masschallenge.atlassian.net/browse/AC-5096):**
- Added /v1/functional_expertise/ (list view)
- Added /v1/functional_expertise/<id>/ (detail view)
- Added functional_expertise field to UserListView and UserDetailView

**Test:**
- setup:
  - make dev;make load-remote-db (initial schema); make grant permissions (v1_clients)
  - Create an application for the user with the permissions.
  - Load [Postman collection](https://www.getpostman.com/collections/0a95773f88df764b15ee), initialize the token
- test:
  - in V1 folder, there are two subfolders (among others): Industry, Functional Expertise
  - For each request/response in Industry, the parallel Functional Expertise Request/Response should behave the same.
  - in V1 folder, open User, and look on User List and User. They should now have "functional_expertise" attribute, that behaves like "additional_industries".
